src/parser/solar.rs

```diff
@@ -473,32 +511,26 @@ impl From<&ItemContract<'_>> for Parent {
     }
 }
 
-/// Convert a [`Span`] to a [`TextRange`]
-fn span_to_text_range(span: Span, source_map: &SourceMap) -> TextRange {
+/// Convert a [`Span`] to an utf8 offset stored in a [`TextRange`]
+/// @dev Only the uft8 offset of the TextRante is initially populated, the rest being filled in `complete_text_indices` (avoid multiple src parsing for the utf16)
```

Comment:
```suggestion
/// Convert a [`Span`] to an utf8 offset stored in a [`TextRange`]
///
/// Only the utf8 offset of the [`TextRange`] is initially populated, the rest being filled via [`complete_text_ranges`] to avoid duplicate work.
```